runelite-client/src/main/java/net/runelite/client/plugins/npcunaggroarea/NpcAggroAreaConfig.java

```diff
@@ -83,21 +84,34 @@ default boolean showAreaLines()
 	}
 
 	@ConfigItem(
-		keyName = "npcUnaggroAreaColor",
-		name = "Area lines colour",
-		description = "Choose colour to use for marking NPC unaggressive area",
+		keyName = "npcAggroAreaColor",
+		name = "Aggressive colour",
+		description = "Choose colour to use for marking NPC unaggressive area when NPCs are aggressive",
 		position = 5
 	)
+	@Alpha
 	default Color aggroAreaColor()
 	{
-		return Color.YELLOW;
+		return new Color(0x64FFFF00, true);
+	}
+
+	@ConfigItem(
+		keyName = "npcUnaggroAreaColor",
+		name = "Unaggressive colour",
+		description = "Choose colour to use for marking NPC unaggressive area after NPCs have lost aggression",
+		position = 6
+	)
+	@Alpha
+	default Color unaggroAreaColor()
+	{
+		return new Color(0xFFFF00);
```

Comment:
Shouldn't this just be Color.YELLOW

Reply:
Imo its more clear for it to match the other config